magda-web-client/src/Components/SearchFacets/SearchFacets.scss

```diff
@@ -51,13 +54,10 @@
     &:hover,
     &:focus {
         background-color: #f7f8fa;
+        color: $main;
     }
 }
-
 .search-facets-mobile {
-    margin: 0 -15px;
-    width: calc(100% + 30px);
-
     .facet-header au-btn {
```

Comment:
should `.facet-header au-btn` be `.facet-header .au-btn` (a dot missing before au-btn) ?

Reply:
yeah thanks for spotting that!